src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -228,17 +226,9 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                         args.Index++;
 
                     }
-                    else if (arg.StartsWith("/Tc"))
-                    {
-                        // TODO completely skipped for now - used in ReactOS only for compilation of resources
-                        files.Clear();
-                        break;
-
-                    }
-                    else if (arg.StartsWith("/Tp"))
+                    else if (arg.StartsWith("/Tc") || arg.StartsWith("/Tp"))
```

Comment:
As discussed, It is safe to treat `/Tc` like `/TC` and `/Tp` like `/TP`

Reply:
Opened an issue: https://github.com/SonarSource/sonarlint-visualstudio/issues/1778